code/modules/mob/living/carbon/human/human_attackhand.dm

```diff
@@ -234,7 +238,14 @@
 				if(org.status & LIMB_DESTROYED)
 					status = "MISSING!"
 
-				to_chat(src, "\t [status=="OK"?"<span class='notice'> ":"<span class='warning'> "]My [org.display_name] is [status].</span>")
+				if(brute_treated == FALSE & brutedamage > 0)
```

Comment:
```suggestion
				if(brute_treated == FALSE && brutedamage > 0)
```
Check the reference for the difference between `&` and `&&`. You'll want all those instances here to be `&&`.